Django/README.md

```diff
@@ -6,16 +6,22 @@ To read more about the recruitment process and open opportunities at Wealth42, c
 
 ### Pick an evaluation mode
 
-1.   **Project of your choice**: You are free to submit a project of your own with *some* conditions.
-     *   Qualifying criteria for submission:
-         *   MUST exhibit everything covered in the [Assessment Criteria](#assessment-criteria) and [Instructions](#instructions).
-         *   You are authorised to publish the code as a public pull request via GitHub
-     *   Follow instructions as in the [Submission Process](#submission-process) to submit your work
-
-2.   **Wealth42's drafted assignment**: 
+1.   **Wealth42's drafted assignment**: 
      *   You can select any one of the stories mentioned in the [stories](../stories/) folder, and develop an API service using Django for it. 
      *   Follow [Instructions](#instructions) and [Submission Process](#submission-process) to complete the assignment.
      *   For success, ensure you get as many checks as in our [Assessment Criteria](#assessment-criteria)
+     *   We typically take 5 days to evaluate this submission.
+
+
+2.   **Project of your choice**: You are free to submit a project of your own with *some* conditions.
+     *   Qualifying criteria for submission:
+         *   MUST exhibit everything covered in the [Assessment Criteria](#assessment-criteria) and [Instructions](#instructions).
+         *   You are authorised to publish the code as a public pull request via GitHub, following the steps as covered in [Submission Process](#submission-process)
+     *   Additional criteria for submission:
```

Comment:
Let's keep the 'attach the story' in the submission process section, Nitish had made it that way on purpose. To see the exact changes Nitish had made, you can check [here](https://github.com/wealth42/nick-fury/commit/8397425b4a97e47e0885b98cb870f2a32653d638). Let's also keep it that way for the rest of the assignments too.
The ordering for 'project of your choice' and 'Wealth42's drafted assignment' is fine though.